.github/workflows/container-image.yml

```diff
@@ -1,51 +1,27 @@
-name: Build porta container image
+name: Container
 on:
-  # push:
-  #   branches:
-  #   - master
-  workflow_dispatch: {}
-  # pull_request:
-  # branches: [ master ]
-
-env:
-  # Use docker.io for Docker Hub if empty
-  REGISTRY: ghcr.io
-  # github.repository as <account>/<repo>
-  IMAGE_NAME: ${{ github.repository }}
+  push:
+    branches:
+      - master
+      - managed-services
+      - 3scale-[0-9]+-stable
```

Comment:
Is `3scale-[0-9]+-stable` a regular expression? In that case it won't match `3scale-2.13-stable` because it doesn't match the point `.`.

Better `3scale-[0-9.]+-stable`, right?

Reply:
This is not a regular expression. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
But you are right. Will update it right away.